test/unit/dialects/postgres/data-types.test.js

```diff
@@ -1,44 +0,0 @@
-'use strict';
-
-const chai = require('chai');
-
-const expect = chai.expect;
-const Support = require('../../support');
-
-const dialect = Support.getTestDialect();
-const { DataTypes: BaseTypes } = require('@sequelize/core');
-const DataTypes = require('@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/postgres/data-types.js')(BaseTypes);
-const { PostgresQueryGenerator: QueryGenerator } = require('@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/postgres/query-generator.js');
-
-if (dialect.startsWith('postgres')) {
-  describe('[POSTGRES Specific] DataTypes', () => {
-    beforeEach(function () {
-      this.queryGenerator = new QueryGenerator({
-        sequelize: this.sequelize,
-        _dialect: this.sequelize.dialect,
-      });
-    });
-
-    describe('GEOMETRY', () => {
```

Comment:
Why are these tests removed?

Reply:
they have been moved outside of the postgres folder, they now run on all dialects that support GEOMETRY/GEOGRAPHY (which is at least postgres, mysql, mariadb & mssql, though this last one is not implemented yet)